internal/io/limitwriter.go

```diff
@@ -0,0 +1,49 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package io provides a LimitWriter that writes to an underlying writer up to
+// a limit.
+
+package io
+
+import "io"
+
+// LimitWriter is a writer that writes to an underlying writer up to a limit.
+type LimitWriter struct {
```

Comment:
Please follow the naming convention of the `io` package. That is, `LimitedWriter` and `LimitWriter()` just like [io.LimitedReader](https://pkg.go.dev/io#LimitedReader) and [io.LimitReader](https://pkg.go.dev/io#LimitReader).

Reply:
You should also follow the similar fields.

Reply:
Updated.